Unit test error in master branch action

<!-- Please state your issue using the following template and, most importantly, in English. -->

#### Steps/Code to reproduce:
```
# github.com/milvus-io/milvus/internal/datanode [github.com/milvus-io/milvus/internal/datanode.test]
internal/datanode/data_node.go:327: assignment mismatch: 1 variable but etcdkv.NewEtcdKV returns 2 values
FAIL	github.com/milvus-io/milvus/internal/datanode [build failed]
FAIL
internal/datanode/data_node.go:327: cannot use etcdClient (type *clientv3.Client) as type []string in argument to etcdkv.NewEtcdKV
internal/datanode/data_node_test.go:340:7: assignment mismatch: 1 variable but etcdkv.NewEtcdKV returns 2 values
internal/datanode/data_node_test.go:340:26: cannot use client (type *clientv3.Client) as type []string in argument to etcdkv.NewEtcdKV
make: *** [test-go] Error 2
Makefile:115: recipe for target 'test-go' failed
```

#### Expected result:


#### Actual results:



#### Environment:
- Milvus version(e.g. v2.0.0-RC2 or 8b23a93):
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others: 

#### Configuration file:


#### Additional context:

--- a/internal/datanode/data_node.go
+++ b/internal/datanode/data_node.go
@@ -320,11 +320,11 @@ func (node *DataNode) Start() error {
 	}
 
 	connectEtcdFn := func() error {
-		etcdClient, err := clientv3.New(clientv3.Config{Endpoints: Params.EtcdEndpoints})
+		etcdKV, err := etcdkv.NewEtcdKV(Params.EtcdEndpoints, Params.MetaRootPath)
 		if err != nil {
 			return err
 		}
-		node.kvClient = etcdkv.NewEtcdKV(etcdClient, Params.MetaRootPath)
+		node.kvClient = etcdKV
 		return nil
 	}
 	err = retry.Do(node.ctx, connectEtcdFn, retry.Attempts(ConnectEtcdMaxRetryTime))

--- a/internal/datanode/data_node_test.go
+++ b/internal/datanode/data_node_test.go
@@ -24,7 +24,6 @@ import (
 	"github.com/golang/protobuf/proto"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"go.etcd.io/etcd/clientv3"
 	"go.uber.org/zap"
 
 	etcdkv "github.com/milvus-io/milvus/internal/kv/etcd"
@@ -333,59 +332,56 @@ func TestWatchChannel(t *testing.T) {
 	defer cancel()
 
 	t.Run("test watch channel", func(t *testing.T) {
-		client, err := clientv3.New(clientv3.Config{Endpoints: Params.EtcdEndpoints})
-		assert.Nil(t, err)
-		if assert.NotNil(t, client) {
 
-			kv := etcdkv.NewEtcdKV(client, Params.MetaRootPath)
-			ch := fmt.Sprintf("datanode-etcd-test-channel_%d", rand.Int31())
-			path := fmt.Sprintf("channel/%d/%s", node.NodeID, ch)
-			c := make(chan struct{})
-			go func() {
-				ec := kv.WatchWithPrefix(fmt.Sprintf("channel/%d", node.NodeID))
-				cnt := 0
-				for {
-					evt := <-ec
-					for _, event := range evt.Events {
-						if strings.Contains(string(event.Kv.Key), ch) {
-							cnt++
-						}
-					}
-					if cnt >= 2 {
-						break
+		kv, err := etcdkv.NewEtcdKV(Params.EtcdEndpoints, Params.MetaRootPath)
+		require.NoError(t, err)
+		ch := fmt.Sprintf("datanode-etcd-test-channel_%d", rand.Int31())
+		path := fmt.Sprintf("channel/%d/%s", node.NodeID, ch)
+		c := make(chan struct{})
+		go func() {
+			ec := kv.WatchWithPrefix(fmt.Sprintf("channel/%d", node.NodeID))
+			cnt := 0
+			for {
+				evt := <-ec
+				for _, event := range evt.Events {
+					if strings.Contains(string(event.Kv.Key), ch) {
+						cnt++
 					}
 				}
-				c <- struct{}{}
-			}()
-
-			vchan := &datapb.VchannelInfo{
-				CollectionID:      1,
-				ChannelName:       ch,
-				UnflushedSegments: []*datapb.SegmentInfo{},
+				if cnt >= 2 {
+					break
+				}
 			}
-			info := &datapb.ChannelWatchInfo{
-				State: datapb.ChannelWatchState_Uncomplete,
-				Vchan: vchan,
-			}
-			val, err := proto.Marshal(info)
-			assert.Nil(t, err)
-			err = kv.Save(path, string(val))
-			assert.Nil(t, err)
+			c <- struct{}{}
+		}()
 
-			<-c
-			node.chanMut.RLock()
-			_, has := node.vchan2SyncService[ch]
-			node.chanMut.RUnlock()
-			assert.True(t, has)
-
-			kv.RemoveWithPrefix(fmt.Sprintf("channel/%d", node.NodeID))
-			//TODO there is not way to sync Release done, use sleep for now
-			time.Sleep(100 * time.Millisecond)
-
-			node.chanMut.RLock()
-			_, has = node.vchan2SyncService[ch]
-			node.chanMut.RUnlock()
-			assert.False(t, has)
+		vchan := &datapb.VchannelInfo{
+			CollectionID:      1,
+			ChannelName:       ch,
+			UnflushedSegments: []*datapb.SegmentInfo{},
 		}
+		info := &datapb.ChannelWatchInfo{
+			State: datapb.ChannelWatchState_Uncomplete,
+			Vchan: vchan,
+		}
+		val, err := proto.Marshal(info)
+		assert.Nil(t, err)
+		err = kv.Save(path, string(val))
+		assert.Nil(t, err)
+
+		<-c
+		node.chanMut.RLock()
+		_, has := node.vchan2SyncService[ch]
+		node.chanMut.RUnlock()
+		assert.True(t, has)
+
+		kv.RemoveWithPrefix(fmt.Sprintf("channel/%d", node.NodeID))
+		//TODO there is not way to sync Release done, use sleep for now
+		time.Sleep(100 * time.Millisecond)
+
+		node.chanMut.RLock()
+		_, has = node.vchan2SyncService[ch]
+		node.chanMut.RUnlock()
+		assert.False(t, has)
 	})
 }

